crates/forge/bin/cmd/bind.rs

```diff
@@ -209,17 +217,43 @@ No contract artifacts found. Hint: Have you built your contracts yet? `forge bin
     fn generate_bindings(&self, artifacts: impl AsRef<Path>) -> Result<()> {
         let bindings = self.get_multi(&artifacts)?.build()?;
         println!("Generating bindings for {} contracts", bindings.len());
+
+        let bindings_root_path = self.bindings_root(&artifacts);
+
         if !self.module {
             trace!(single_file = self.single_file, "generating crate");
             bindings.dependencies([r#"serde = "1""#]).write_to_crate(
                 &self.crate_name,
                 &self.crate_version,
-                self.bindings_root(&artifacts),
+                &bindings_root_path,
                 self.single_file,
-            )
+            )?;
+
+            self.update_cargo_toml(bindings_root_path.as_path())?;
         } else {
             trace!(single_file = self.single_file, "generating module");
-            bindings.write_to_module(self.bindings_root(&artifacts), self.single_file)
+            bindings.write_to_module(&bindings_root_path, self.single_file)?;
         }
+
+        Ok(())
+    }
+    fn update_cargo_toml(&self, bindings_root_path: &Path) -> Result<()> {
+        let cargo_toml_path = bindings_root_path.join("Cargo.toml");
+        let mut cargo_toml = std::fs::OpenOptions::new()
+            .append(true)
+            .open(cargo_toml_path)
+            .wrap_err("Failed to open Cargo.toml for appending")?;
+
+        if let Some(description) = &self.crate_description {
+            writeln!(cargo_toml, "description = {:?}", description)
+                .wrap_err("Failed to write description to Cargo.toml")?;
+        }
```

Comment:
this is not that easy because this will append the description to the end of the file.
I think we want this just under the `rust-version` version field
I don't think we want to fix this in ethers, but I think we can just find the first empty line in the cargo.toml and put the description and license in there

Reply:
ty for your review, 

> this is not that easy because this will append the description to the end of the file. I think we want this just under the `rust-version` version field I don't think we want to fix this in ethers, but I think we can just find the first empty line in the cargo.toml and put the description and license in there

so if i understand well what you means, i need to put those value just after rust-version, right ? 

```
[package]
name = "bot"
version = "0.1.0"
edition = "2021
rust-version = "1.56"

```

Reply:
Also what if the cargo.tmol doesn't contain the rust-version field, should i return error like "rust-version not found in Cargo.toml"

Reply:
okey so i'm trying to found rus(version in the cargo file but if it doesn't found the rust version, it fail. Is that what we are trying to do here?